homeassistant/components/ssdp/__init__.py

```diff
@@ -401,7 +413,7 @@ async def _ssdp_listener_callback(
             return
 
         discovery_info = discovery_info_from_headers_and_description(info_with_desc)
-        discovery_info[ATTR_HA_MATCHING_DOMAINS] = matching_domains
+        discovery_info[ATTR_HA_MATCHING_DOMAINS] = matching_domains  # type: ignore [typeddict-item]
```

Comment:
Can we get rid of the ignore comment?  I think it's ok to include it in the typeddict since we expect integrations to be able to use it

Reply:
I managed to resolve this using new type: `_SsdpServiceInfoBase`